recipes/perl-class-accessor/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+schema_version: 1
+
```

Comment:
```suggestion
```
We don't want these lines in the recipe, they are leftovers from my automatic conversion